lib/utapi/utilities.js

```diff
@@ -82,11 +98,15 @@ export function listBucketMetrics() {
         .option('-v, --verbose')
         .parse(process.argv);
 
-    const { host, port, accessKey, secretKey, start, end,
-        buckets, verbose, ssl } =
+    const { host, port, accessKey, secretKey, start, end, verbose,
+        ssl } =
         commander;
-    const requiredOptions = { host, port, accessKey, secretKey, buckets,
-        start };
+    const requiredOptions = { host, port, accessKey, secretKey };
+
+    // Ensure the proper metric is provided.
+    const metric = metricType === undefined ? commander.metric : metricType;
```

Comment:
Okay, I updated to first check for `commander.metric` as a required argument and then added a comment and more clarity to how we are defining `metric` here.